cdap-ui/app/cdap/components/DataPrepConnections/NoDefaultConnection/index.tsx

```diff
@@ -54,7 +53,7 @@ const NoDefaultConnection: React.SFC<INoDefaultConnectionProps> = ({
           browserTitle={T.translate(`${PREFIX}.title`).toString()}
         />
 
-        <EmptyMessageContainer title={T.translate(`${PREFIX}.title`)}>
+        <EmptyMessageContainer title={T.translate(`${PREFIX}.title`).toString()}>
```

Comment:
@yukiej `T.translate` returns a `ReactNode`. Is it always guaranteed to convert it to string? IMHO it would make sense to set the type of `title` to be `ReactNode` since that gets rendered in DOM.